tests/components/huawei_lte/test_switches.py

```diff
@@ -16,7 +16,7 @@
 
 from tests.common import MockConfigEntry
 
-SWITCH_WIFI_GUEST_NETWORK = "switch.lte_wifi_guest_network"
+SWITCH_WIFI_GUEST_NETWORK = "switch.lte_wi_fi_guest_network"
```

Comment:
This is curious, does it change as result of these modifications? If it does, would it not be a breaking change?

Reply:
Nope, since the unique id isn't changed